packages/common/db_schema.py

```
@@ -0,0 +1,226 @@
+
+        except Exception as e:
+            # Rollback on error
+            conn.rollback()
+            logger.error(
+                "Schema creation failed, rolling back",
+                extra={
+                    "correlation_id": correlation_id,
+                    "error": str(e),
+                    "error_type": type(e).__name__,
+                },
+            )
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)